### PR TITLE
Added multi URL run functionality

### PR DESCRIPTION
**Pull request by atyachari**

### CODE REVIEW

File: multipleLighthouseRun.js [hunk 1]
```diff
@@ -1,36 +1,43 @@
...
+
+switch(urlSourceType) {
+  case URLSource.File:
+    handleFileCase({urlSource, runLimit, platform, otherParameters, result})
+    break;
+  case URLSource.URL:
+    handleURLCase({urlSource, runLimit, platform, otherParameters, result})
+    break;
+}
+if(urlSourceType == URLSource.File) {
```
Comment (anantnema) [inline multipleLighthouseRun.js]: remove this redundant if else

File: multipleLighthouseRun.js [hunk 2]
```diff
@@ -1,36 +1,43 @@
...
+let urlSource = argument[2] || "https://www.google.com" ;
 let runLimit = argument[3] || 1;
 let platform = argument[4] || "desktop";
 
 let otherParameters = argument.slice(5, argument.length).join(" ");
 
-validatingParameters({ url, runLimit, platform });
-
-console.log(
-  "\x1b[36m%s\x1b[0m",
```
Comment (anantnema) [inline multipleLighthouseRun.js:23]: Add this log in URL and for file both

**Comment by anantnema** [comment]
LGTM

**Review by anantnema — APPROVED** [review]

File: multipleLighthouseRun.js [hunk 3]
```diff
@@ -1,36 +1,35 @@
...
+  validatingParametersAndFetchURLSourceType,
+  URLSourceType,
+  handleFileCase,
+  handleURLCase
+} = require("./utilities.js");
 
 const argument = process.argv;
-let url = argument[2] || "https://www.google.co.in";
-let runs = 0;
+let urlSource = argument[2] || "https://www.google.com" ;
```
Comment (punam555) [inline multipleLighthouseRun.js]: can hardcoded urls move to a constant section?

**Review by punam555 — APPROVED** [review]
approved.

**Pull request merged**